archive/tarfile_test.go

```diff
@@ -0,0 +1,13 @@
+package archive
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestWriteToTar(t *testing.T) {
```

Comment:
Nothing to address: We have been using the test capability just to run our code from our package. We should start writing some actual unit tests to go with the code